lib/node-sftp-s3.js

```diff
@@ -255,7 +255,7 @@ class SFTPServerStorage extends EventEmitter {
                 state.read = true;
               }
 
-				  this.storage.getObject(state.fullname, `bytes=${offset}-${offset+length-1}`, function(err, data) {
+              this.storage.getObject(state.fullname, `bytes=${offset}-${offset+length-1}`, (err, data) => {
```

Comment:
I would love to have test for this, but I'm not sure if it's going to be straightforward; probably will tinker with it myself in spare time :). I assume you tested it and it solves the problem that you have, so I'll merge and release this.